src/openmrs-backend-dependencies.ts

```diff
@@ -1,10 +1,12 @@
 import { openmrsFetch } from "@openmrs/esm-api";
 import * as semver from "semver";
-import { difference, isEmpty } from "lodash-es";
+import { difference, isEmpty, intersection } from "lodash-es";
 
 var installedBackendModules: any[] = [];
 
 const originalOnload = System.constructor.prototype.onload;
+let modulesWithWrongVersionInstalled: MismatchedModuleVersion[] = [];
```

Comment:
These should not be global variables. Find a way to export modules with their respective dependencies.